backend/typescript/rest/schedulingRoutes.ts

```diff
@@ -101,6 +103,30 @@ schedulingRouter.get("/volunteers/:volunteerId?", async (req, res) => {
   }
 });
 
+schedulingRouter.get("/pickup/:isPickUp", async (req, res) => {
+  const { isPickUp } = req.params;
+  const contentType = req.headers["content-type"];
+
+  console.log("pickup", isPickUp);
+
+  if (typeof isPickUp !== "string") {
```

Comment:
should check that it's "true" or "false" here as well, otherwise passing in gibberish to `isPickup` will return us `getSchedulingsByPickUp(false)`